packages/dds/tree/README.md

```diff
@@ -151,7 +151,20 @@ When it is more designed, some details for how it works belong in this section (
 
 ### Editing
 
-TODO: add a diagram for this section.
+Edit related data flow with solid arrows.
+Key view related updates made in response with dotted arrows.
+
+This shows editing during a transaction:
+
+```mermaid
+graph RL;
```

Comment:
I'm also wondering if this particular graph would benefit from being a sequence diagram instead. The arrows are positioned in a way that makes them a bit confusing and it's difficult to tell which actors in this process matter.

Reply:
I tried a sequence diagram and I have mixed feelings about it. I'll make a separate PR for that change so its easier to compare

Reply:
I think with the subgraphs, it's probably not as necessary.